zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -585,6 +660,7 @@ where
                 ));
 
                 // Return immediately if the chain tip has changed.
+                latest_chain_tip.mark_best_tip_seen();
```

Comment:
I think this method may be marking the best tip as *unseen* (decrementing the watch channel receiver instance's version so the next call to `changed` is immediately ready).

Reply:
Ah you're right, I didn't read the method docs, and caused this bug myself. I'll go revert those changes.

Reply:
Fixed in 8656076e39